### PR TITLE
Homework 1 Submission

### PR DESCRIPTION
**Pull request by narrelleg**
I used RMarkdown to write a brief bio with text emphasis, bullet points, a heading, a link, and an image. I followed the steps in lab to fork and clone the repository so that I could make these changes in RStudio, and then sync them back to Git with stage, commit, and finally a pull request. The process was fairly easy because of the clear instructions, helpful online guide, and detailed tutorial in the lab. Markdown seems to be very user friendly, though I did have one issue where my bullet points did not work. I eventually realized that I needed to add an extra line break. Git seems less transparent because the names of operations (eg. fork, clone) do not make their function obvious, but I hope to become very familiar with them all soon.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# About Me
+
+My name is **Narrelle Gilchrist**. I am an incoming *Committee on International Relations* student at UChicago, and I am excited to learn how to use R for political science research. Here are some quick facts about me:
+
+* I am from Florida, and earned a BA from the **University of Florida**.
```
Comment (maddiherne) [inline README.md:7]: Good use of interesting formatting techniques here!

File: README.md [hunk 2]
```diff
@@ -1 +1,14 @@
...
+  + My BA is in Political Science and International Studies.
+* My research interests focus on political institutions in sub-Saharan Africa.
+* In my free time, I like to watch Star Trek, write science fiction, and sing in choirs.
+
+For more, you can [look at my LinkedIn.](https://www.linkedin.com/in/narrelle-gilchrist)
```
Comment (maddiherne) [inline README.md:12]: Linked-in was a good link to share here! Looks good!

**Review by maddiherne — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       X    |              |            |
| **Coding strategy**         |       X    |              |            |
| **Presentation**            |      X     |              |            |
| **Achievement, creativity** |      X     |              |            |
| **Ease of access**          |    X       |              |            |

Remarks:

* Simple, but good quality work!
* Good job hitting ever criterium required!

**Review by rrodriguezbarron — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Excellent assignment overall
* Great use of bullets with levels
* URL link worked
* Star Trek is a great series... have you watched Battlestar Galactica?

**Pull request closed without merging**